src/ralph/backends/database/es.py

```diff
@@ -0,0 +1,66 @@
+"""Elasticsearch storage backend for Ralph"""
+
+import json
+import logging
+import sys
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import scan, streaming_bulk
+
+from .base import BaseDatabase
+
+logger = logging.getLogger(__name__)
+
+
+class ESDatabase(BaseDatabase):
+    """Elasticsearch database backend"""
+
+    name = "es"
```

Comment:
If you want to explore more OOP, you can declare a `@property` method returning this value ? You are sure this value will never change and you can also declare it as an abstract method in the abstract class. WDYT ?

Reply:
It's a class attribute. It is not supposed to change.

Reply:
well maybe but nothing prevent you to a change of this attribute.